ipatests/test_integration/test_commands.py

```diff
@@ -710,3 +710,58 @@ def test_samba_config_file(self):
             ['ipa-adtrust-install', '-a', 'Secret123', '--add-sids', '-U'])
         res = self.master.run_command(['testparm', '-s'])
         assert 'ERROR' not in (res.stdout_text + res.stderr_text)
+
+    def test_sss_ssh_authorizedkeys(self):
+        """Login via Ssh using private-key for ipa-user should work.
+
+        Test for : https://pagure.io/SSSD/sssd/issue/3937
+        Steps:
+        1) setup user with ssh-key and certificate stored in ipaserver
+        2) simulate p11_child timeout
+        3) try to login via ssh using private key.
+        """
+        user = 'testsshuser'
+        passwd = 'Secret123'
+        user_key = tasks.create_temp_file(self.master, create_file=False)
+        pem_file = tasks.create_temp_file(self.master, directory='/tmp')
```

Comment:
`directory` argument is not needed here, the default directory for `mktemp` is /tmp